Add issue template for pip works, uv fails cases (#1986)
astral-sh/uv: .github/ISSUE_TEMPLATE/pip-works--uv-doesn-t.md

---
name: Pip works, uv doesn't
about: For cases where `pip install` works but `uv pip install` fails
title: ''
labels: ''
assignees: ''

---

**Description**

> <!-- If you're using a non-standard python setup, please include some context on your setup. -->

Reviewer: I feel like people who know that their Python setup is non-standard will probably already put that information in an issue without this prompt, and people who don't know that their Python setup is non-standard probably won't understand what's meant by the term "non-standard" here... Maybe it's better to just ask some concrete questions here? Suggestions:
- How did you install Python? (python.org, pyenv, Windows store, conda, Linux distro, etc.)
- Please give the output of running `python -VV` on the command line

Author: Those should be covered by the last two lines, i've removed this and left it as a blanket "description" header

Reviewer: makese sense!

**Requirements**

> <!-- Please provide a list of requirements (requirements.in or requirements.txt), ideally a minimal set that fails. -->

Reviewer: ```suggestion
<!-- Please provide a list of the requirements you attempted to install with uv (requirements.in or requirements.txt), ideally a minimal set that fails. -->
```

```

```

**pip command and output**

> <!-- The command you use to install with pip. Please make sure you are using `--no-cache-dir` to disable using cached built wheels. You can link long output as a [gist](https://gist.github.com/). -->

Reviewer: ```suggestion
<!-- The working pip command. Please make sure you are using `--no-cache-dir` to disable using cached built wheels. You can link long output as a [gist](https://gist.github.com/). -->
```

```shell
pip install --no-cache-dir <your_options_here>
```

**uv command**

> <!-- The command you use to install with uv. Please make sure you are using `--no-cache`. -->

Reviewer: ```suggestion
<!-- The uv command you tried. Please make sure you are using `--no-cache`. -->
```
